roxie/roxiemem/roxiemem.cpp

```diff
@@ -169,6 +173,28 @@ static void initializeHeap(bool allowHugePages, bool allowTransparentHugePages,
 #else
     heapUseHugePages = false;
 
+#ifdef _USE_NUMA
+    bool has_numa = numa_available() >= 0;
+    if (has_numa)
+    {
+        struct bitmask *nodemask = numa_get_mems_allowed();
+        if (nodemask && *nodemask->maskp)
+        {
+            int serr = set_mempolicy(MPOL_INTERLEAVE, nodemask->maskp, nodemask->size);
```

Comment:
Should it be possible to configure this via an option - even if it is only to easily test what effect it has?